packages/styles/scss/patterns/sub-patterns/content-section/_content-section.scss

```diff
@@ -30,16 +30,33 @@
       }
     }
     &__heading {
-      margin-top: $carbon--layout-01;
+      margin-top: $carbon--layout-03;
       margin-bottom: $carbon--layout-05;
 
       @include type-style('expressive-heading-02');
+
+      @include carbon--breakpoint('md') {
+        margin-top: $carbon--layout-03;
+      }
+      @include carbon--breakpoint('lg') {
+        margin-top: $carbon--layout-01;
+      }
```

Comment:
Why are you changing all of this? I think you had it correct in the first place. The issue doesn't talk about needing to fix this either.

Reply:
I changed because 'sm' breakpoint does nothing so did like above

Reply:
I'm not sure I understand what you're saying. There is no `sm` breakpoint mixin here. You added `md` and `lg`. You now have too much spacing at the `sm` and `md` breakpoints. It looks like it's supposed to be `32px` and now based on this code (still waiting for it build) it looks like you have `48px`.